articles/mfa/guides/customize-phone-messages.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Customize SMS or Voice Messages
+topics:
+  - mfa
+  - guardian
+  - sms
+contentType:
+  - how-to
+useCase:
+  - customize-mfa
+---
+# Customize SMS or Voice Messages
+
+To customize the SMS or Voice messages sent by Auth0 during enrollment or verification, do the following:
+
+First, go to the [Multi-factor Auth page](${manage_url}/#/mfa), then click on the **Phone** box to configure your Phone Messaging settings.
+
+![MFA Phone Settings](/media/articles/mfa/sms-settings.png)
+
+You have two fields to customize your messages:
+* **Enrollment Template**: the message sent by Auth0 during enrollment.
+* **Verification Template**: the message sent by Auth0 during authentication.
+
+[Liquid](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) syntax is the supported templating engine to use when accessing user attributes in SMS templates. The following attributes are available:
+
+* `message_type`: Can be “sms” or “voice”, and can be used to detect which kind of message it's being sent.
```

Comment:
`Can be “sms” or “voice”,` it's using these weird quote character probably copied from google doc, let's replace it with the standard quotes `"` instead of `“`.